### ISSUE TITLE
MS 2010 importing

### ISSUE THREAD

**Issue by Integral** [substance/archivist-composer#80]
@oliver----, please take a look at http://ost.d4s.io/editor#55927409293a700a00a5d741
It's importing MS WORD styles inside document.

**Comment by obuchtala** [comment]
@Integral Is that only happening when pasting from 2010?
I don't have this version... how can we reproduce it?
Do you have this version?
We need a way to log the HTML coming in from the clipboard, then I can fix it.

**Comment by Integral** [comment]
No, I don't have it. How could we log it? Via console?

**Comment by obuchtala** [comment]
Can you monkey patch the clipboard.js at this position: https://github.com/substance/substance/blob/055e3d01178a5ae2fe776cc389ea15a1e4c9c4fe/src/surface/clipboard.js#L126
Adding

```
console.log('#####################################')
console.log(html);
console.log('#####################################')
```

and then let somebody paste it...

Is that possible?

**Comment by Integral** [comment]
Yes, it is possible... Let's wait with it a bit...Till commit. I'll paste
it here later. Thanks.
